src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -294,19 +292,16 @@ function deleteQuestion(number) {
  * Allows users to cancel editing questions
  */
 function cancelEdit(number) {
-    if (number === -1) {
-        if (confirm('Are you sure you want to cancel adding this question?')) {
-            location.reload();
-        }
-    } else {
-        if (confirm('Are you sure you want to cancel your changes?')) {
-            location.reload();
-        }
+    var confirmationMsg = number === -1 ?
+            'Are you sure you want to cancel adding this question?' :
+            'Are you sure you want to cancel your changes?';
```

Comment:
add parenthesis around `number === -1`?
The line wrapping does not follow the Java convention. Is it different for JavaScript?